docs/sphinx/source/api.rst

```diff
@@ -379,7 +378,8 @@ Effects on PV System Output
 ===========================
 
 Loss models
-^^^^^^^^^^^
+-----------
```

Comment:
`Loss models` is the only heading under `Effects on PV System Output`.  Should the functions be brought up a level?

Reply:
I added more subheadings below.